EnglishFilter/sections/antiadblock.txt

```diff
@@ -572,6 +572,8 @@ uaclips.com,kzclip.com,ru-clip.com,ruclip.com###morev
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/78539
+@@||filmisub.com/templates/filmisub/js/fuckadblock.min.js$domain=filmisub.com
```

Comment:
It's a Bulgarian site like `filmiseriali.com`.
You can exchange those rules to one `@@/fuckadblock.min.js$domain=filmiseriali.com|filmisub.com`.